src/simd_nf4.h

```diff
@@ -168,9 +168,15 @@ inline __uint128_t pack(__uint128_t a, uint32_t flag)
 
 #if defined(__AVX2__)
 
-inline VecType CAST_TO_DOUBLE(HalfVecType x)
+inline VecType load_to_reg(HalfVecType x)
```

Comment:
the `_a` and `_b` can probably be const, no?

Also, try to avoid prefixing name with `_`, this is usually reserved for the compiler and internals.